internal/handlers/common/aggregations/operators/accumulators/accumulators.go

```diff
@@ -22,14 +22,17 @@
 package accumulators
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/FerretDB/FerretDB/internal/handlers/commonerrors"
 	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/iterator"
+	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
 // newAccumulatorFunc is a type for a function that creates an accumulation operator.
-type newAccumulatorFunc func(expression *types.Document) (Accumulator, error)
+type newAccumulatorFunc func(args ...any) (Accumulator, error)
```

Comment:
It is possible to reason about the old signature and documentation comment (that function accepts an aggregation expression as a document). It is not possible to reason about the new signature that accepts any amount of any arguments without updating the documentation comment.

Please also don't skip the checklist.